config.py

```diff
@@ -2556,6 +2592,35 @@ def get_fs_subjects_dir():
                 .resolve())
 
 
+def get_all_contrasts() -> Iterable[ArbitraryContrast]:
```

Comment:
I suggest to do the validation here

Reply:
OK, I can see why and I'm happy to move it there. My reasoning was that moving the checks in `get`ters would duplicate the logic, as we have two getters, but I could abstract away a `_validate_contrasts(...)` to be called from both.